### DISCUSSION TITLE
Multiple Trackers?

### DISCUSSION THREAD

**Discussion by Symbolics** [google/docsy#650] [Q&A]
Does anyone know if you can put multiple trackers onto a site?

**Background**:

When I set up Google Analytics, I accepted all the defaults. Being new at this, I didn't really understand the differences between UA and GA, and figured they would be about the same. Google defaulted to GA4.

Whilst this may not be a surprise to many, GA4 is quite difficult to work with. For example I spent nearly six hours yesterday trying to accomplish a simple task: find the URLs of the referrers. It took me a while to notice that all the examples I found on the 'net were using a different system altogether: UA. And, as strange as it may seem, there appears to be no way to get a referrers URL in GA4! Knowing that a user came from "wordpress.com" or "github.com" isn't very useful; we need to know specifically where it came from.

So, I'm thinking we need to move to UA, but there's some folks using GA. Push come to shove, we can just force a move and lose all the existing data if we have to, but I thought I'd see what others suggest for (free) tracking on netlify and if the two Google solutions can co-exist. (Or, possibly, there is a way to get the referrer URL that's very well hidden)/

**Comment by LisaFC** [comment]
Hrmm, I am not an analytics expert (and the bits I do know about aren't related to referrer URLs) - I did find this discussion which might be helpful? The first answer isn't useful as I think the poster was looking at the wrong UI but there's further discussion lower down. https://www.quora.com/Why-does-Google-Analytics-GA4-no-longer-show-referral-URLs-like-Universal-Analytics-UA-In-UA-youd-go-to-Acquisition-All-Traffic-Referrals-GA4-at-Acquisition-Traffic-Acquisition-only-shows-referring-Domain-but-not

Also it looks like you can tag sites for both UA and GA, which might help too: https://developers.google.com/analytics/devguides/collection/ga4/basic-tag?technology=analyticsjs
